src/main/resources/beanRefContext.xml

```diff
@@ -51,18 +51,89 @@
   </bean>
 
   <bean id="metrics" class="ome.system.metrics.NullMetrics"/>
+  
+  <bean name="/OMERO/Pixels2" class="com.glencoesoftware.omero.ms.image.region.PixelsService"
+        parent="filesystem">
+    <!-- index=0 "path" comes from parent -->
+    <constructor-arg index="1" ref="MemoizerWait"/>
+    <constructor-arg ref="omeroFilePathResolver"/>
+    <constructor-arg ref="backOff"/>
+    <constructor-arg ref="tileSizes"/>
+    <constructor-arg><null /></constructor-arg>
+    <property name="metrics" ref="metrics"/>
+  </bean>
+
+  <bean id="iscale" class="ome.logic.JavaImageScalingService"/>
 
   <bean id="omero-ms-image-region-verticle"
         class="com.glencoesoftware.omero.ms.image.region.ImageRegionVerticle"
         scope="prototype">
-    <constructor-arg ref="/OMERO/Pixels" />
+    <constructor-arg ref="/OMERO/Pixels2" />
```

Comment:
Any particular reason for this change?

Reply:
Previously these had used the PixelsService in ms-core, and that bean is still available under that name. ImageRegionVerticle doesn't use this class anymore, but I can rename the bean to something better anyway